### ISSUE TITLE
[javascript] InaccurateNumericLiteral underscore separator notation false positive

### ISSUE THREAD

**Issue by tysonstewart** [pmd/pmd#4165]
**Affects PMD Version:** 6.48.0

**Rule:** InaccurateNumericLiteral

Please provide the rule name and a link to the rule documentation:
<https://pmd.github.io/latest/pmd_rules_ecmascript_errorprone.html#innaccuratenumericliteral>

**Description:** JavaScript containing numeric literals with [underscore separators](https://github.com/tc39/proposal-numeric-separator) triggers the rule.

**Code Sample demonstrating the issue:**

```javascript
const value = 1_000_000;
```

**Expected outcome:**

PMD reports a violation but this is an integer. It's a false positive.

**Running PMD through:** *Codacy*
